jhu-idc/idc-isle-dc
# Fixes the export FieldFormatters; fixes 20-export-tests migrations

Resolves: https://github.com/jhu-idc/iDC-general/issues/389

**This updates the snapshot to give all the current content unique ids, so export can work.**  

This essentially catches export up with all the recent `field_unique_id` changes. 

- The FieldFormatters were out of date and export was not exporting the data in the correct format with the correct quad default. It now does. 
- Adds `util.js` and therefore `doMigration` to the 20-export-tests.  Now the migrations are checked to ensure they are passing. 

To test: 

- check out this branch and make reset
- run the migrations from 20-export-test (or whichever migrations you'd like). 
- run an export: `https://islandora-idc.traefik.me/export_items?query=(ss_type:islandora_object)`
- compare the exported CSV to the migration CSV
    - The export should look, content-wise, like the imported data - but you'll notice it will use the field_unique_id by default.  It will also change the order of values in multivalue fields sometimes.
